src/utils/limiter.ts

```diff
@@ -6,10 +6,7 @@ import {
 import { logger } from './logger.js';
 import { BlockfrostServerError } from '@blockfrost/blockfrost-js';
 
-export const pLimiter = new PQueue({ concurrency: BLOCKFROST_REQUEST_CONCURRENCY });
-
-// separate queue for assets' metadata as these will be mostly memoized
-export const assetMetadataLimiter = new PQueue({ concurrency: BLOCKFROST_REQUEST_CONCURRENCY / 5 });
+export const pLimiter = new PQueue({ concurrency: BLOCKFROST_REQUEST_CONCURRENCY || 1 });
```

Comment:
One would expect default to be 500 due to
```
export const BLOCKFROST_REQUEST_CONCURRENCY = Number(
  process.env.BLOCKFROST_REQUEST_CONCURRENCY ?? 500,
);
```

But I get it, if there is NaN then it would fallback to 1, but wouldn't it be less confusing to fallback to the same default value?

Reply:
`|| 1` is not a default, it is required for _unlimited requests_: calling `new PQueue({ concurrency: 0 })` throws error.

Reply:
ahh I meant fallback, not default. I assumed it is for handling `NaN`. So if env `BLOCKFROST_REQUEST_CONCURRENCY` is set to some invalid value (string `"aa"`) then concurrency will be set to 1, which is a bit confusing. I would expect it fallbacks to the default value of 500.

This implementation feels like workaround since the limiter instance is not even used when `BLOCKFROST_REQUEST_CONCURRENCY` is set to 0.

To keep things simple I suggests we completely omit the ability to set concurrency to unlimited